File: solutions/go/etl/1/etl.go
package etl

import "strings"

func Transform(in map[int][]string) map[string]int {
	result := make(map[string]int)
	
	for score, letters := range in {
		for _, letter := range letters {
			// Convert the letter to lowercase and add to result map
			result[strings.ToLower(letter)] = score
		}
	}
	
	return result
}

File: solutions/cpp/etl/1/etl.cpp
#include "etl.h"
#include <cctype>

namespace etl {

    std::map<char, int> transform(std::map<int, std::vector<char>> old) {
        std::map<char, int> result;
        for (const auto& [score, letters] : old) {
            for (char letter : letters) {
                result[std::tolower(letter)] = score;
            }
        }
        return result;
    }

}  // namespace etl